docs/sources/release-notes/v2-0.md

```diff
@@ -0,0 +1,205 @@
+querier:
+    max_concurrent_queries: 20
+    search:
+        prefer_self: 10
+ingester:
+    concurrent_flushes: 4
+    max_block_duration: 30m
+    max_block_bytes: 524288000
+storage:
+    trace:
+        pool:
+            max_workers: 400
+            queue_depth: 20000
+        search:
+            read_buffer_count: 32
+            read_buffer_size_bytes: 1048576
+
+```
+
+### Removed and renamed configuration parameters
+
+The following tables describe the parameters that have been removed or renamed. (PR#1978)
+
+#### Removed
+
+| Parameter | Comments |
+| :-- | :-- |
+| query_frontend:<br/>  query_shards: |   |
+| querier:<br/>  query_timeout: | Use `trace_by_id.query_shards` |
+| ingester:<br/>  use_flatbuffer_search: | Automatically set based on block type |
+| storage:<br/>  wal:<br/>    version: | Removed and pinned to `block.version` |
+
+#### Renamed
+
+| Parameter | Comments |
+| --- | --- |
+| `compactor` section |  |
+| compaction:<br/>  chunk_size_bytes: | Renamed to `v2_in_buffer_bytes` |
+| compaction:<br/>  flush_size_bytes: | Renamed to `v2_out_buffer_bytes` |
+| compaction:<br/>  iterator_buffer_size: | Renamed to `v2_prefetch_traces_count` |
+| `storage` section |  |
+| wal:<br/>  encoding: | Renamed to `v2_encoding` |
+| block:<br/>  index_downsample_bytes: | Renamed to `v2_index_downsample_bytes` |
+| block:<br/>  index_page_size_bytes: | Renamed to `v2_index_page_size_bytes` |
+| block:<br/>  encoding: | Renamed to `v2_encoding` |
+| block:<br/>  row_group_size_bytes: | Renamed to `parquet_row_group_size_bytes` |
+
+
+### Other upgrade considerations
+
+[PR 2004](https://github.com/grafana/tempo/pull/2004) The  `search_enabled` and `metrics_generator_enabled` configuration parameters have been removed. Both default to true.
+
+[PR 1879](https://github.com/grafana/tempo/pull/1879). Use snake case on Azure Storage config. Here is an example of using snake case on Azure Storage config:
+
+```yaml
+# config.yaml
+storage:
+  trace:
+    azure:
+      storage_account_name:
+      storage_account_key:
+      container_name:
+```
+
+[PR 1678](https://github.com/grafana/tempo/pull/1678). Parquet is the new default block version. To continue using the v2 backend set, see the [Parquet configuration documentation](https://grafana.com/docs/tempo/latest/configuration/parquet/).
+
+[PR 1810](https://github.com/grafana/tempo/pull/1810). Delete `TempoRequestErrors` alert from mixin. Any Jsonnet users relying on this alert should copy this into their own environment.
+
+[PR 1794](https://github.com/grafana/tempo/pull/1794). metrics-generator: Handle collisions between user defined and default dimensions. Custom dimensions colliding with intrinsic dimensions will be prefixed with `__`.
+
+[PR 1754](https://github.com/grafana/tempo/pull/1754). Add TLS support to the vulture. Internal types are updated to use `scope` instead of `instrumentation_library`. This is a breaking change in trace by ID queries if JSON is requested.
```

Comment:
```suggestion
[PR 1754](https://github.com/grafana/tempo/pull/1754). We've added TLS support to the vulture. Internal types are updated to use `scope` instead of `instrumentation_library`. This is a breaking change in "trace by ID" type queries if JSON is requested.
```